test/jdk/java/lang/Thread/ThreadStopTest.java

```diff
@@ -22,60 +22,43 @@
  */
 
 /* @test
- * @bug  8289610 8249627 8205132
- * @summary Test that Thread stop/suspend/resume throw UOE
- * @run junit DegradedMethodsThrowUOE
```

Comment:
You didn't really need to rename the test even though there is now only one degraded method left.

Reply:
> You didn't really need to rename the test even though there is now only one degraded method left.

True but it's wasn't good name for a test that only exercises one method.